src/nft-card-back.ts

```diff
@@ -181,18 +207,24 @@ export class NftCardBackTemplate extends LitElement {
     `
   }
 
-  @property({type: Object}) public traitData = {}
+  @property({type: Object}) public traitData!: TraitData
```

Comment:
what are you trying to do with the exclamation point? it just suppresses typescript's nullability checks, but some of these properties do look nullable

Reply:
Basically these are properties passed down by the `nft-card` parent component. These values will be defined by the parent that is why I added the `!`. For instance, if `traitData` is not defined then that means we are either loading or there is an error. `nft-card-front` will not render in either of these 2 cases.